src/commands/convert/prowler2hdf.ts

```diff
@@ -2,26 +2,44 @@ import {Command, Flags} from '@oclif/core'
 import fs from 'fs'
 import {ASFFResults as Mapper} from '@mitre/hdf-converters'
 import {checkSuffix} from '../../utils/global'
+import _ from 'lodash'
+import path from 'path'
 
 export default class Prowler2HDF extends Command {
-  static usage = 'convert prowler2hdf -i <asff-finding-json> [--securityhub <standard-1-json> ... <standard-n-json>] -o <hdf-scan-results-json>'
+  static usage = 'convert prowler2hdf -i <prowler-finding-json> -o <hdf-output-folder>'
 
   static description = 'Translate a Prowler-derived AWS Security Finding Format results from concatenated JSON blobs into a Heimdall Data Format JSON file'
 
-  static examples = ['saf convert prowler2hdf -i prowler-asff.json -o output-hdf-name.json']
+  static examples = ['saf convert prowler2hdf -i prowler-asff.json -o output-folder']
 
   static flags = {
     help: Flags.help({char: 'h'}),
-    input: Flags.string({char: 'i', required: true}),
-    output: Flags.string({char: 'o', required: true}),
+    input: Flags.string({char: 'i', required: true, description: 'Input Prowler ASFF JSON file'}),
+    output: Flags.string({char: 'o', required: true, description: 'Output folder'}),
   }
 
   async run() {
     const {flags} = await this.parse(Prowler2HDF)
     // comes as an asff-json file which is basically all the findings concatenated into one file instead of putting it in the proper wrapper data structure
```

Comment:
learned that this type of structure is actually a JSON Lines / jsonl format.  

https://jsonlines.org/

```suggestion
    // comes as an asff-json file (their naming scheme for a JSON Lines/jsonl format file) which needs to be converted into a normal json object
```